src/hooks/wallet/useLedger.ts

```diff
@@ -0,0 +1,44 @@
+import { Ledger } from '@polkadot/hw-ledger';
+import { useAccount } from 'src/hooks/useAccount';
+import { useStore } from 'src/store';
+import { computed, ref, watch } from 'vue';
+
+enum LedgerId {
+  nanoS = 'nanoS',
+  nanoSPlus = 'nanoSP',
+  nanoX = 'nanoX',
+}
+
+export const useLedger = () => {
+  const isLedgerAccount = ref<boolean>(false);
+  const isLedgerNanoS = ref<boolean>(false);
+  const { currentAccount } = useAccount();
+
+  const store = useStore();
+  const isH160 = computed<boolean>(() => store.getters['general/isH160Formatted']);
+
+  const handleReset = (): void => {
+    isLedgerAccount.value = false;
+    isLedgerNanoS.value = false;
+  };
+
+  const handleLedgerData = async (): Promise<void> => {
+    if (!currentAccount.value || currentAccount.value === null || isH160.value) {
+      handleReset();
+      return;
+    }
+    try {
+      const ledger = new Ledger('hid', 'astar');
```

Comment:
I got this popup when I **haven't** connected Ledger to my laptop (which is annoying for users). I'm not sure how to remove it but I'll try to find a solution...

![image](https://github.com/AstarNetwork/astar-apps/assets/92044428/0849c067-cf8a-413b-b0a6-f4bd0d1f5a17)

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/765/commits/b04843f5b4a0e055e5ed7006b09e19b3dfdb0076) commit